install-dev/data/xml/hook.xml

```diff
@@ -4254,5 +4254,10 @@
       <title>Modify title grid template data</title>
       <description>This hook allows to modify data which is about to be used in template for title grid</description>
     </hook>
+    <hook id="actionJsAfterModuleImport">
+      <name>actionJsAfterModuleImport</name>
+      <title>Add JS after module import</title>
+      <description>This hook allows to add a JS function to execute after a module import. This will help to perform actions after module upgrade in a different process</description>
```

Comment:
```suggestion
      <description>This hook allows to add a JS function to execute after a module import. This will help to perform actions after module upgrade in a different process (this hook is temporary and ill be removed in the next major)</description>
```